package.json

```diff
@@ -36,26 +36,25 @@
     "version": "npm run build && git add -A dist"
   },
   "dependencies": {
-    "es6-error": "^3.2.0",
-    "es6-promise": "^4.0.4",
-    "kinvey-node-sdk": "3.3.5",
-    "lodash": "^4.16.2",
-    "parse-headers": "^2.0.1",
-    "rxjs": "5.0.0-beta.12",
```

Comment:
@thomasconner I see rxjs in the `webpack.config` but removed from the `package.json`. Wouldn't this affect developers that use npm with this SDK?